tools/packaging/static-build/shim-v2/build.sh

```diff
@@ -23,6 +23,19 @@ container_image="${SHIM_V2_CONTAINER_BUILDER:-$(get_shim_v2_image_name)}"
 
 EXTRA_OPTS="${EXTRA_OPTS:-""}"
 
+if [ "${MEASURED_ROOTFS}" == "yes" ]; then
+	EXTRA_OPTS+=" DEFSERVICEOFFLOAD=true"
+	info "Enable rootfs measurement config"
+
+	root_hash_file="${repo_root_dir}/tools/osbuilder/root_hash.txt"
+	[ -f "$root_hash_file" ] || \
```

Comment:
I guess for the near future (and in `CCv0` now) we might have `${repo_root_dir}/tools/osbuilder/root_hash_tdx.txt` for TDX instead of `root_hash.txt`, so we'll need to update the logic to fail if none of the root hash files are found then.

Reply:
Yeah - I think you are right that doing the backport straight away (well after we have the CCv0 CI stable 😭 ) is the best approach rather than have it mixed in with the rest of the merged changes